config.go

```diff
@@ -1159,252 +1123,147 @@ func ValidateConfig(cfg Config, interceptor signal.Interceptor, fileParser,
 			"mutually exclusive, only one should be selected")
 	}
 
-	// Determine the active chain configuration and its parameters.
-	switch {
-	// At this moment, multiple active chains are not supported.
-	case cfg.Litecoin.Active && cfg.Bitcoin.Active:
-		str := "Currently both Bitcoin and Litecoin cannot be " +
-			"active together"
-		return nil, mkErr(str)
-
-	// Either Bitcoin must be active, or Litecoin must be active.
-	// Otherwise, we don't know which chain we're on.
-	case !cfg.Bitcoin.Active && !cfg.Litecoin.Active:
-		return nil, mkErr("either bitcoin.active or " +
-			"litecoin.active must be set to 1 (true)")
-
-	case cfg.Litecoin.Active:
-		err := cfg.Litecoin.Validate(
-			minTimeLockDelta, funding.MinLtcRemoteDelay,
-		)
-		if err != nil {
-			return nil, mkErr("error validating litecoin: %v", err)
-		}
-
-		// Multiple networks can't be selected simultaneously.  Count
-		// number of network flags passed; assign active network params
-		// while we're at it.
-		numNets := 0
-		var ltcParams chainreg.LitecoinNetParams
-		if cfg.Litecoin.MainNet {
-			numNets++
-			ltcParams = chainreg.LitecoinMainNetParams
-		}
-		if cfg.Litecoin.TestNet3 {
-			numNets++
-			ltcParams = chainreg.LitecoinTestNetParams
-		}
-		if cfg.Litecoin.RegTest {
-			numNets++
-			ltcParams = chainreg.LitecoinRegTestNetParams
-		}
-		if cfg.Litecoin.SimNet {
-			numNets++
-			ltcParams = chainreg.LitecoinSimNetParams
-		}
-		if cfg.Litecoin.SigNet {
-			return nil, mkErr("litecoin.signet is not supported")
-		}
-
-		if numNets > 1 {
-			str := "The mainnet, testnet, and simnet params " +
-				"can't be used together -- choose one of the " +
-				"three"
-			return nil, mkErr(str)
-		}
-
-		// The target network must be provided, otherwise, we won't
-		// know how to initialize the daemon.
-		if numNets == 0 {
-			str := "either --litecoin.mainnet, or " +
-				"litecoin.testnet must be specified"
-			return nil, mkErr(str)
-		}
-
-		// The litecoin chain is the current active chain. However
-		// throughout the codebase we required chaincfg.Params. So as a
-		// temporary hack, we'll mutate the default net params for
-		// bitcoin with the litecoin specific information.
-		chainreg.ApplyLitecoinParams(&cfg.ActiveNetParams, &ltcParams)
-
-		switch cfg.Litecoin.Node {
-		case "ltcd":
-			err := parseRPCParams(
-				cfg.Litecoin, cfg.LtcdMode,
-				chainreg.LitecoinChain, cfg.ActiveNetParams,
+	// For backwards compatibility, the cfg.Bitcoin.Active option is still
+	// exposed to the user, but it now must be set to true since no other
+	// chains are supported.
+	if !cfg.Bitcoin.Active {
```

Comment:
nit: maybe we should mark it deprecated?

Reply:
mark it deprecated and then just ignore the actual value? or still read the actual value for now? 

Ie, do we mark it deprecated but then still require that they set it?

Reply:
good q...I think we can ignore the value

Reply:
cool - done!